build-release

```diff
@@ -440,6 +440,7 @@ build () {
 			done
 		fi
 
+		local strip=''
```

Comment:
Why not doing `strip='strip'` by default in a way we drop the later `else` to rely on the default on non-specific use case?

Reply:
I was thinking of doing the opposite - to remove `local` from all variables in the `build` function - because of something I discovered: `local` breaks `set -e` error propagation. if you do `x=$(false)`, the script exits. But with `local x=$(false)`, it keeps going.

Reply:
Oops, replied to the wrong comment.